daemon/images/image.go

```diff
@@ -161,12 +161,12 @@ func (i *ImageService) GetImage(ctx context.Context, refOrID string, options ima
 		}
 		p := *options.Platform
 		// Note that `platforms.Only` will fuzzy match this for us
-		// For example: an armv6 image will run just fine an an armv7 CPU, without emulation or anything.
+		// For example: an armv6 image will run just fine an armv7 CPU, without emulation or anything.
```

Comment:
This probably had to be "on an armv7 CPU"

```
// For example: an armv6 image will run just fine on an armv7 CPU, without emulation or anything.
```

Reply:
thanks! fixed!